### PR TITLE
Feature/create html css

### PR DESCRIPTION
**Pull request by Logandv3**
## Description

Please include a summary of the change and/or functionality implementation:
This pull request creates the basic structure for the main page, recipe grid and individual recipe that will appear when a recipe is clicked on.  It also applies basic styling to the page.  This also creates the first query selectors, event listeners and event handlers.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] open index.html

## Checklist:

- [x] I have performed a self-review of my own code
- [x] No console error messages

## Additional info
Please include any other relevant information here:
Try out Elvis Pancakes

### CODE REVIEW

**Review by EricSergeant — APPROVED** [review]
Good organization of event listeners and functions on scripts.js file.  Solid outline for multiple functionality to be completed soon.

File: src/index.html [hunk 1]
```diff
@@ -1,10 +1,89 @@
...
+          </section>
+          <section class="grid-item">
+            <div class="card-head">
+              <p>Elvis Pancakes</p>
+            </div>
+            <div class="card-body">
+              <img src="https://spoonacular.com/recipeImages/741603-556x370.jpeg" alt="Elvis Pancakes">
+            </div>
+          </section>
+          <section class="grid-item"></section>
```
Comment (elisebeall) [inline src/index.html:63]: Placeholders for recipe cards that will be populated by javascript

File: src/scripts.js [hunk 2]
```diff
@@ -1,6 +1,87 @@
...
+  // Issue:  Need multiple tags.
+
+  // Check if user has entered a name or ingredient.  If so prioritize that and
+  // run searchRecipes fcn.
+
+  // If there is no value entered in search bar check for tags selected and search
+  // by tags.
+
+  // If neither is selected do nothing.
+};
```
Comment (elisebeall) [inline src/scripts.js:68]: Great notes to help with writing the functionality

File: src/styles.css [hunk 3]
```diff
@@ -1,3 +1,57 @@
+* {
+  border: 1px solid black;
+}
+
 body {
-  background-color: blueviolet;
-}
```
Comment (elisebeall) [inline src/styles.css:3]: 🥲

**Pull request merged**